package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
How exactly is `imports[1]` being loaded above and what is it? I couldn't actually quite work that out.

From what I can tell, this is because `react-transform-catch-errors` expects module objects to be components? While ES modules are always namespaces. 

If you want everything to work without effort, I'd advise not using jspm.